docs/modules/ROOT/pages/how-to-guides/Import-code/proc_importing_code.adoc

```diff
@@ -0,0 +1,207 @@
+//[id="proc_importing_code_{context}"]
+
+= Importing and configuring code
+
+[role="_abstract"]
+After you click **Create an application**, the **Grab some code** page opens. From here, choose one of the following source code options to begin building your application:
+
+* **Bring in your own code**
+* **Start with a sample**
+
+== Using a code sample
+
+. Optional: In the code sample **Search** field, begin typing the name of the code you want to work with. If {ProductName} has code samples in that language stack, the code sample results filter accordingly.
+. Optional: If you see a language stack or framework that you want to work with, click either its **Open Git repository** or **Import sample** link.    
+* If you click **Open Git repository**, a new tab opens to the GitHub repository for the code framework you chose.
+* If you click **Import sample**, {ProductName} creates and deploys a new application with the sample code you chose. Check out the open **Overview** tab to monitor your continuous integration/continuous deployment (CI/CD) activity.
+
+.What's next
+**Configure your components for deployment** opens. From here, configure your components before you deploy them. You can also review or change any values that {ProductName} populated for you. When you're finished here, you're one click away from creating your application.
```

Comment:
This section doesn't apply to samples, only the bring your own code path. Once they click import, there's a spinner, and then they land on their new application with a build running. Samples skip configuration since we know the basics on how to set it up reliably.